src/Illuminate/Support/helpers.php

```diff
@@ -195,7 +195,7 @@ function array_has($array, $key)
      * @param  mixed  $default
      * @return mixed
      */
-    function array_last($array, $callback, $default = null)
+    function array_last($array, $callback = null, $default = null)
```

Comment:
`$callback` should also be type hinted to `callable` for consistency with `array_first`.